django-backend/fecfiler/transactions/serializers.py

```diff
@@ -274,6 +274,7 @@ def to_representation(self, instance):
                 }
             )
 
+        representation["can_delete"] = instance.can_delete
```

Comment:
I think you can get rid of this line and add `can_delete = BooleanField(read_only=True)` to the fields above.  that will give us all the features of a serializer field